components/library/libgdiplus/test/results-all.master

```diff
@@ -0,0 +1,73 @@
+make[1]: Verzeichnis „$(@D)“ wird betreten
```

Comment:
Could you add the transform rule to process the test results? Probably you could just add
`USE_DEFAULT_TEST_TRANSFORMS=yes`
since I added the switch in https://github.com/OpenIndiana/oi-userland/commit/8b5563981dc141c1d4ac07d6d516ac6351499d50